modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm

```diff
@@ -320,7 +358,6 @@
 /obj/item/storage/box/gunset/glock18_hos
 	name = "GK-18 supply box"
 	desc = "Ideally contains a fast-firing 9x19mm pistol made out of cheap plastic."
-	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
It was done this way as it otherwise doesn't pass CI anymore, since you're trying to put a bulky in a bulky, which doesn't work.

Reply:
Fixed.  However, this seems strange that HoS is the only head who spawns with their gun.